src/main/thrift/parquet.thrift

```diff
@@ -191,6 +191,74 @@ enum FieldRepetitionType {
   REPEATED = 2;
 }
 
+/**
+  * A histogram of repetition and definition levels for either a page or column
+  * chunk.
+  *
+  * This is useful for:
+  *   1. Estimating the size of the data when materialized in
+  *   memory
+  *
+  *   2. For filter push-down on nulls at various levels of nested
+  *   structures and list lengths.
+  */
+struct RepetitionDefinitionLevelHistogram {
+   /**
+    * When present, there is expected to be one element corresponding to each
+    * repetition (i.e. size=max repetition_level+1) where each element
```

Comment:
why +1? Shouldn't this have one element if `repetition_level == 1`?

Reply:
if max-rep-level == 1,  it could has rep==1 and rep==2.

Reply:
I guess 0 and 1, but yeah, true, my bad. Makes sense.